src/ogdc_runner/models/recipe_config.py

```diff
@@ -36,8 +36,8 @@ class RecipeConfig(BaseModel):
     # allowable characters in k8s object names. `id` to construct such names.
     id: str = Field(..., pattern=r"^[a-z0-9.-]+$")
 
-    # Type of recipe, e.g., "simple", "visualization", etc.
-    type: Literal["simple", "visualization"] = Field(default="simple")
+    # Type of recipe, e.g., "shell", "visualization", etc.
+    type: Literal["shell", "visualization"] = Field(default="shell")
```

Comment:
I actually think we should probably remove the default here and force recipe authors to choose. This is a little more explicit and ensures that authors are aware different types of recipes exist.

Reply:
Created an issue to address here: https://github.com/QGreenland-Net/ogdc-runner/issues/77